package.json

```diff
@@ -9,7 +9,7 @@
   "license": "MIT",
   "repository": "jshttp/accepts",
   "dependencies": {
-    "mime-types": "^3.0.0",
+    "mime-types": "^3.0.1",
```

Comment:
This would already be installed, was there a security issue with 3.0.0 that required this?

Reply:
No security issues, we said we would not normally need to do these (anymore), but since it will not be updated as a transitive if the applications have a lock file, and the PR was already open, it didn't make sense to not merge it.